_integration-test/custom-ca-roots/setup.sh

```diff
@@ -25,8 +25,12 @@ openssl req -new -nodes -newkey rsa:2048 -keyout $TEST_NGINX_CONF_PATH/self.test
 
 # openssl req -in nginx/self.test.req -text -noout
 
+# Store extension details in a temporary file since python subprocess can't use process substitution
+temp_file=$(mktemp)
+printf "subjectAltName=DNS:self.test" >"$temp_file"
+
```

Comment:
oh this shouldn't change at all -- this is failing because the file is missing a shebang `#!/usr/bin/env bash` and is using bash features.  the os will default to `sh` which doesn't understand the original syntax

Reply:
I did try this locally, but it still didn't work. Let me try it in CI to see if it makes a difference

Reply:
Looks like it did not work https://github.com/getsentry/self-hosted/actions/runs/8330451282/job/22795132410?pr=2892

Reply:
the calling code is also wrong -- it's using `.` which is an alias to `source` (instead of executing it) which isn't going to do what you expect anyway:  https://youtu.be/YE6ZDpRcdQU

Reply:
calling code corrected, now it's working :)